src/main/java/mate/academy/rickandmorty/services/DataLoader.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.rickandmorty.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.external.CharacterDto;
+import mate.academy.rickandmorty.dto.external.CharacterResponseDto;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.models.Characters;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class DataLoader {
+    private static final String BASE_URL = "https://rickandmortyapi.com/api/character";
```

Comment:
You can get it from properties
```suggestion
    @Value("${rick-and-morty.url}")
    private String baseUrl;
```